Fix badge circle

BenjaminZekavica opened the pull request:
Reference: https://wordpress.org/five-for-the-future/pledge/benjamin-zekavica-projects/ 
I fixed the Circle CSS for displaying the badges and spacing on the single page. 

**Before:** 

![before](https://github.com/WordPress/five-for-the-future/assets/15845265/1988f557-698b-4982-9ae2-0dd29bdf67e7)


**After:**

![after](https://github.com/WordPress/five-for-the-future/assets/15845265/c8605cb6-5751-45a8-a238-b1733c81703e)

BenjaminZekavica committed:
fix badge width and height

BenjaminZekavica committed:
formatting

BenjaminZekavica commented:
Can anyone check the CI? Thanks :)

StevenDufresne commented:
> Can anyone check the CI? Thanks :)

Unrelated to this PR. 👍

BenjaminZekavica commented:
@StevenDufresne Perfect, thank you :) Do you can approve this PR?

iandunn commented:
Hey Benjamin, thanks for the PR 👋🏻 I'm not sure what problem this is solving, though. Can you explain that more?

BenjaminZekavica commented:
@iandunn You're welcome :) I fixed the circle, because if the text on the right side is bigger, than the circle looks like an egg :D 

I added and tested the new css Code and it works very well.  

![Bildschirmfoto 2023-10-18 um 21 07 03](https://github.com/WordPress/five-for-the-future/assets/15845265/b9c97dda-79a5-42ef-9953-6d22a337ca23)

iandunn commented:
Ah, I sse it now 👍🏻

iandunn reviewed (approved):
LGTM 👍🏻

iandunn commented:
It's the end of my day, so I'm going to wait until tomorrow to sync/deploy this.

BenjaminZekavica commented:
> It's the end of my day, so I'm going to wait until tomorrow to sync/deploy this.

Thank you so much :)
